apps/payments/app/[locale]/(hosted)/paymentRequest/stripe/StripeHost.tsx

```
@@ -0,0 +1,45 @@
+"use client";
+import React, { useMemo } from "react";
+import {
+  Appearance,
+  StripeElementsOptions,
+  loadStripe,
+} from "@stripe/stripe-js";
+import { Elements } from "@stripe/react-stripe-js";
+import CheckoutForm from "./CheckoutForm";
+
+type Props = {
+  clientSecret?: string;
+  returnUri: string;
+};
+
+const PUBLISHABLE_KEY = process.env.NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY ?? "";
```

Shall we throw if `NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY` is undefined instead of using an empty string?

---

[9432c9b](https://github.com/ogcio/life-events/pull/1/commits/9432c9b6ed0ddd93a2d32c5c6c6493763a34adc6)